src/main/java/org/terasology/logic/health/BlockDamageRenderer.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2019 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.logic.health;
+
+import com.google.common.collect.ArrayListMultimap;
+import com.google.common.collect.Multimap;
+import org.terasology.entitySystem.entity.EntityManager;
+import org.terasology.entitySystem.entity.EntityRef;
+import org.terasology.entitySystem.systems.BaseComponentSystem;
+import org.terasology.entitySystem.systems.RegisterMode;
+import org.terasology.entitySystem.systems.RegisterSystem;
+import org.terasology.entitySystem.systems.RenderSystem;
+import org.terasology.math.geom.Vector3i;
+import org.terasology.registry.In;
+import org.terasology.rendering.assets.texture.Texture;
+import org.terasology.rendering.assets.texture.TextureRegionAsset;
+import org.terasology.rendering.world.selection.BlockSelectionRenderer;
+import org.terasology.utilities.Assets;
+import org.terasology.world.block.BlockComponent;
+import org.terasology.world.block.regions.BlockRegionComponent;
+
+import java.util.Optional;
+
+/**
+ * This system renders damage damaged blocks using the BlockSelectionRenderer.
+ */
+@RegisterSystem(RegisterMode.CLIENT)
+public class BlockDamageRenderer extends BaseComponentSystem implements RenderSystem {
+
+    private BlockSelectionRenderer blockSelectionRenderer;
+
+    @In
+    private EntityManager entityManager;
+
+    @Override
+    public void renderOverlay() {
+        if (blockSelectionRenderer == null) {
+            Texture texture = Assets.getTextureRegion("health:blockdamageeffects#1").get().getTexture();
```

Comment:
Even though the [BlockDamageEffects](https://github.com/Terasology/Health/pull/4/files#diff-ec2d137ee6854b1fa1fbab10fb27efef) has been moved into the module, this line is creating `NoSuchElementException`.